src/v2-router.js

```diff
@@ -777,10 +813,62 @@ v2Router.get('/public-key', async (request, response) => response.status(501).se
 /**
  * Send a login link to a visitor.
  */
-v2Router.post('/registrations/:id/login', async (request, response) =>
+v2Router.get('/registrations/:id/login', async (request, response) =>
   response.status(501).send({message: 'Not implemented.'})
 );
 
+/**
+ * Send out a renewal email to user if email has been found in database.
+ */
+v2Router.post('/registrations/renewalEmailCheck', async (request, response) => {
+  // Check that the visitor's given us an email address.
+  const {email} = request.query;
+  const emailInvalid = email === undefined;
+
+  // Check if there's a registration allocated at the specified email address.
+  const emailExists = await Registration.findAllEmails(email);
```

Comment:
Might want to `try catch` this call as if `email` is undefined the API crashes :)

Reply:
Resolved in https://github.com/Scottish-Natural-Heritage/trap-registration-api/pull/124/commits/5c55bc20ba9e89291e7dcc8936715a64d6809c49.